src/core/net/private/geoipdatabase.cpp

```diff
@@ -40,28 +36,15 @@
 #include "core/types.h"
 #include "geoipdatabase.h"
 
-struct Node
-{
-    quint32 left;
-    quint32 right;
-};
-
-struct GeoIPData
-{
-    // Metadata
-    quint16 ipVersion;
-    quint16 recordSize;
-    quint32 nodeCount;
-    QDateTime buildEpoch;
-    // Search data
-    QList<Node> index;
-    QHash<quint32, QString> countries;
-};
-
 namespace
```

Comment:
You could get rid of this **namespace** now (because we have no file-visible classes now). Just change **const** to **static const**.